blockmanager.go

```diff
@@ -2077,6 +2077,13 @@ func (b *blockManager) handleNotifyMsg(notification *blockchain.Notification) {
 		}
 
 		for _, stx := range block.STransactions()[0:] {
+			// Remove rebroadcast inventory for revocations
+			if txType := stake.DetermineTxType(stx.MsgTx()); txType == stake.TxTypeSSRtx {
```

Comment:
This section is incorrect in at least 2 ways:

* A block being connected does not invalidate a revocation.  It's only invalidated if it was a revocation for a ticket which was previously invalid, but is now valid again
* It's skipping the required mempool updates below due to the continue